### ISSUE TITLE
Alias "browse" as "open"

### ISSUE THREAD

**Issue by elia** [github/hub#782]
I get this wrong all the time, probably the reason is that other CLI adopted "open" to perform similar actions. Example that come to mind:

- `open`
- `bundle open`
- `gem open`
- `powder open`

Of course I know* that I can alias it in my `.gitconfig`, but maybe it will come useful to others.

_* I just found it works

**Comment by jingweno** [comment]
I agree it may make sense to rename it to `open` for the same reason. But maybe this could be done after 2.2.0.

**Comment by mislav** [comment]
That's not a bad idea. I just hope git core doesn't add a command called `open`.

**Comment by elia** [comment]
hopefully even if they'll do the behavior will be pretty much the same…

Also https://github.com/paulirish/git-open by the good @paulirish is [the first result I got after searching for "git open"](https://duckduckgo.com/?q=%22git+open%22&t=osx&ia=about)
